### PR TITLE
Change new record selector to be Swift

### PR DESCRIPTION
**Pull request by miburf**
Change so the new record selector page is in Swift, to show mixing Objective-C and swift, and demonstrate using HealthVault methods from swift

### CODE REVIEW

File: Samples/SDKFeatures/SDKFeatures/MHVRecordListViewController.swift
Comment (mikenevi) [inline Samples/SDKFeatures/SDKFeatures/MHVRecordListViewController.swift]: nit: typo

**Review by mikenevi — APPROVED** [review]

File: Samples/SDKFeatures/SDKFeatures/SDKFeatures-Bridging-Header.h
Comment (namalu) [inline Samples/SDKFeatures/SDKFeatures/SDKFeatures-Bridging-Header.h]: I cleaned up the headers in mu most recent pull request, if you wait until my PR is merged, you can just import MHVLib.h

**Review by namalu — APPROVED** [review]

**Pull request merged**
